### PR TITLE
Latest container build should use quay.io registry

### PR DESCRIPTION
**Pull request by hallelujah**
Otherwise we will be limited by docker rate limits

```
Could not pull base image: API error (500): toomanyrequests: You have reached your pull rate limit. You may increase the limit by authenticating and upgrading: https://www.docker.com/increase-rate-limit
```

### CODE REVIEW

File: openshift/system/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM centos/ruby-25-centos7
+FROM quay.io/centos/ruby-25-centos7
```
Comment (gsaslis) [inline openshift/system/Dockerfile:1]:
```suggestion
FROM quay.io/centos7/ruby-25-centos7
```

**Review by gsaslis — CHANGES_REQUESTED** [review]
i think there's a `7` missing there

File: openshift/system/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM centos/ruby-25-centos7
+FROM quay.io/centos7/ruby-25-centos7
```
Comment (gsaslis) [inline openshift/system/Dockerfile:1]:
```suggestion
FROM registry.access.redhat.com/ubi7/ruby-27
```

**Review by gsaslis — CHANGES_REQUESTED** [review]
Better to switch to the official ruby 2.7 image as it will also get security updates and comes from a trusty source.

File: openshift/system/Dockerfile (hunk 2 above)
Comment (hallelujah) [reply openshift/system/Dockerfile:1]: Then we will need to modify a bit the Dockerfile but yes

Comment (akostadinov) [reply openshift/system/Dockerfile:1]: Another option would be to start with clean UBI image and install `rh-ruby25` in build or is the package gone already?

Comment (akostadinov) [reply openshift/system/Dockerfile:1]: But not sure how UBI on RHEL7 works now. I think the following repos need to be enabled.

- rhel-7-server-rpms
- rhel-7-server-optional-rpms
- rhel-7-server-extras-rpms
- rhel-server-rhscl-7-rpms

Or maybe only the last one?

Comment (hallelujah) [reply openshift/system/Dockerfile:1]: We can't use UBI images or we should use 3rd party packages and then there is no point on using the UBI image anymore compared to centos

**Review by gsaslis — APPROVED** [review]
Let's Get This Merged !

**Pull request merged**